backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -422,6 +423,12 @@ func (ctx *AppContext) GetStakeInfo() (*wallettypes.GetStakeInfoResult, error) {
 	return response, nil
 }
 
+// GetColdWalletExtPub performs the rpc command GetColdWalletExtPub.
+func (ctx *AppContext) GetColdWalletExtPub() string {
+	response := ctx.ColdWalletExtPub
+	return response
```

Comment:
Since `ctx.ColdWalletExtPub` is exported, is there really a need for `ctx.GetColdWalletExtPub()`?
It does fall in line with the existing pattern for implementing rpc commands. If keeping this function, can just `return ctx.ColdWalletExtPub` instead of creating and assigning `response` variable.